.github/workflows/compliance.yml

```diff
@@ -31,4 +31,5 @@ jobs:
         # and this very icinga-notifications are licensed under GPL-2.0.
         # https://github.com/Icinga/icingadb/blob/v1.1.1/.github/workflows/compliance/check-licenses.sh
         go-licenses check github.com/icinga/icinga-notifications/... \
-          --allowed_licenses BSD-2-Clause,BSD-3-Clause,GPL-2.0,MIT,MPL-2.0
+          --allowed_licenses BSD-2-Clause,BSD-3-Clause,GPL-2.0,MIT,MPL-2.0 \
+          --ignore github.com/davecgh/go-spew/spew
```

Comment:
> Since we now utilise `testing.T, require.**` in a file that does not end with `*_test.go`, the go licence compliance failed due to the spew package, so I had to put this on the ignore list.

Please don't do that, that would prevent us from noticing possible future license changes of that module and also from accidentally using it in non-test code in the future.

If there was indeed a license problem, it would be better to just avoid the package in this case. However, [it's licensed under the ISC license](https://github.com/davecgh/go-spew/blob/master/LICENSE) which [is compatible with the GPL](https://www.gnu.org/licenses/license-list.en.html#ISC). As that's pretty much the only reason for this check, it can simply be added to the allowed licenses.

Reply:
Done!